wordpress-mobile/WordPress-iOS
# Empty Stats: Reader discover nudge

Fixes [#17126](https://github.com/wordpress-mobile/WordPress-iOS/issues/17126)

This PR adds Reader discover nudge logic and UI.

## Test happy path
1. Go to **My Site** > **Stats** for site with less than 30 views
2. Complete or dismiss the first two nudges (**Publicize** and **Blogging reminders**) in order to have the **Reader discover** nudge shown
3. Tap on **Discover blogs to follow**
4. **Follow topics** screen should be presented
5. Select one or more topics and tap **Done**
6. **Reader** tab should be presented, with **Discover** tab selected

## Test unhappy path
1. Repeat the steps **1, 2, 3, 4** from the happy path
2. Don't select any topic and tap **Done**
3. The **Follow topics** screen should just be dismissed

## Regression Notes
1. Potential unintended areas of impact
n/a

2. What I did to test those areas of impact (or what existing automated tests I relied on)
n/a

3. What automated tests I added (or what prevented me from doing so)
n/a

PR submission checklist:
